test/export/test_swap.py

```diff
@@ -377,12 +377,15 @@ class Foo(torch.nn.Module):
             def forward(self, x, *, inputs):
                 return x + torch.matmul(inputs.a, inputs.b)
 
-        ep = export(
-            Foo(),
-            (torch.randn(2, 2),),
-            {"inputs": CustomInput(torch.randn(2, 3), torch.randn(3, 2))},
-            strict=self.strict,
-        )
+        for use_new_tracer in [True, False]:
+            ep = torch.export._trace._export(
+                Foo(),
+                (torch.randn(2, 2),),
+                {"inputs": CustomInput(torch.randn(2, 3), torch.randn(3, 2))},
+                strict=self.strict,
+                pre_dispatch=True,
+                _use_new_tracer_experimental=use_new_tracer,
```

Comment:
why cant this just call export?

Reply:
oh i think i was just debugging and forgot to change it back.